### PR TITLE
PA-508: Add test to resize volume post restore from locked bucket

### PR DESCRIPTION
**Pull request by kshithijiyer-px**
**What this PR does / why we need it**:
- [x] Add testcase to resize volume post restore from locked bucket
- [x] Add logging err post `GetRuleUid()` call 
- [x] Add `kubectlExec()` to exec and run cmd. 
- [x] Add `getSizeOfMountPoint()` to fix available mountpoint size.

Dashboard URL : http://aetos.pwx.purestorage.com/resultSet/testSetID/77204 

**Which issue(s) this PR fixes** (optional)
Closes # PA-508

**Special notes for your reviewer**:

### CODE REVIEW

**Review by snigdha-px — APPROVED** [review]
LGTM

File: tests/backup/backup_test.go [hunk 1]
```diff
@@ -3934,6 +3935,180 @@ var _ = Describe("{ResizeOnRestoredVolume}", func() {
...
+					postRuleUid, _ := Inst().Backup.GetRuleUid(orgID, ctx, postRuleNameList[0])
+					backupName := fmt.Sprintf("%s-%s-%s", BackupNamePrefix, namespace, backupLocationName)
+					backupList = append(backupList, backupName)
+					CreateBackup(backupName, SourceClusterName, backupLocationName, backupLocationUID, []string{namespace},
+						labelSelectors, orgID, clusterUid, preRuleNameList[0], preRuleUid, postRuleNameList[0], postRuleUid, ctx)
+				})
+				Step("Restoring the backups application", func() {
+					ctx, err := backup.GetAdminCtxFromSecret()
+					log.FailOnError(err, "Fetching px-central-admin ctx")
+					CreateRestore(fmt.Sprintf("%s-restore", backupName), backupName, nil, SourceClusterName, orgID, ctx)
```
Comment (vilasd) [inline tests/backup/backup_test.go:4063]: Should you do validateapplications post restore ? What if the restore fails ?

File: tests/backup/backup_test.go [hunk 2]
```diff
@@ -3934,6 +3935,180 @@ var _ = Describe("{ResizeOnRestoredVolume}", func() {
...
+							log.InfoD(fmt.Sprintf("validate successful volume size increase on app %s's volumes: %v",
+								ctx.App.Key, appVolumes))
+								for _, v := range requestedVols {
+									// Need to pass token before validating volume
+									params := make(map[string]string)
+									if Inst().ConfigMap != "" {
+										params["auth-token"], err = Inst().S.GetTokenFromConfigMap(Inst().ConfigMap)
+										log.FailOnError(err, "Failed to get token from configMap")
+									}
+									err := Inst().V.ValidateUpdateVolume(v, params)
```
Comment (vilasd) [inline tests/backup/backup_test.go:4088]: Helper to check if the volume what application is using is really expanded by execing into pod and checking the volume size

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

File: tests/backup/backup_test.go (hunk 1 above)
Comment (kshithijiyer-px) [reply tests/backup/backup_test.go:4072]: Done

File: tests/backup/backup_test.go (hunk 2 above)
Comment (kshithijiyer-px) [reply tests/backup/backup_test.go:4105]: Done

**Comment by kshithijiyer-px** [comment]
Dashboard URL Post Check add: http://aetos.pwx.purestorage.com/resultSet/testSetID/77687

**Comment by mkoppal-px** [comment]
Add a log.infoD and dash.VerifyFatal when you verify the volume resize on the resulting volumes

**Review by mkoppal-px — APPROVED** [review]
Solid change 👍

**Pull request merged**